src/applets/icon-tasklist/DesktopHelper.vala

```diff
@@ -74,14 +74,6 @@ public class DesktopHelper : GLib.Object {
 		return screen.get_active_window();
 	}
 
-	/**
-	* get_app_launcher will return the last past of an app_id string. Useful when handling the full path to a DesktopAppInfo
-	*/
-	public string get_app_launcher(string app_id) {
-		string[] parts = app_id.split("/");
-		return parts[parts.length - 1];
-	}
-
```

Comment:
Would still like this moved to DesktopHelper in case we need to use it elsewhere.

Reply:
Sure